timbermill-python/timbermill/timberlog.py

```diff
@@ -13,8 +13,8 @@
 thread_instance = local()
 
 
-def init(timbermill_hostname: str, env: str = None, logger=None):
-    timberlog_event_handler.init(timbermill_hostname, env, logger)
+def init(timbermill_hostname: str, env: str = None, logger=None, static_event_params={}):
```

Comment:
@shimk52 default mutable argument is not a good practice: https://docs.python-guide.org/writing/gotchas/#:~:text=Python's%20default%20arguments%20are%20evaluated,to%20the%20function%20as%20well.

Reply:
I can check it inside the method and if None to assign an empty dict to it, but look at all the other functions, `__handle_new_task` for example (and many more), Maybe we should change the dict for all of them? I'm afraid this will create a redundant code to init params in the start of each method.
@aperfectpoint what do you think?

Reply:
I think boy-scout rule and never introduce something that we know is wrong - let's fix it here for now